src/RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -97,6 +98,11 @@ private void RecordTransferHistory(StoreView snapshot, UInt160 scriptHash, UInt1
             transferIndex++;
         }
 
+        private bool IsNull(VM.Types.StackItem item)
+        {
+            return item == null || item.IsNull;
```

Comment:
I think it is impossible for it to be `null`.

Reply:
Yes, we never send `null` as state